### PR TITLE
Analytics Common Lab: Data Actions (Workshop tenancy folder is for testing purposes)

### PR DESCRIPTION
**Pull request by NagwangGyamtso**
# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [ ] All filenames are lower case (including folders, images, files, etc.)
- [ ] Filenames are descriptive
- [ ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ ] Image references in markdown contain an alternate text

### CODE REVIEW

**Comment by william-masdon-oracle** [comment]
Please include the WMS ID of your workshop in the PR

**Comment by NagwangGyamtso** [comment]
These are common labs and don’t have WMS IDs.

[Oracle]<http://www.oracle.com/>
Nagwang Gyamtso
Product Manager, Product Strategy, Analytics
LinkedIn: Linkedin.com/in/nagwanggyamtso/<https://www.linkedin.com/in/nagwanggyamtso/>
Mobile: +1.646.637.3997
Oracle Analytics

From: William Masdon ***@***.***>
Date: Wednesday, January 25, 2023 at 4:59 PM
To: oracle-livelabs/analytics-ai ***@***.***>
Cc: Nagwang Gyamtso ***@***.***>, Author ***@***.***>
Subject: [External] : Re: [oracle-livelabs/analytics-ai] Analytics Common Lab: Data Actions (Workshop tenancy folder is for testing purposes) (PR #99)

Please include the WMS ID of your workshop in the PR

File: common-labs/data-actions/data-actions.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+# Data Actions
+
+## Introduction
+
+In this lab, you will build two simple canvases and learn how to create data actions to pass values between different canvases in a workbook.
+
+  ![Data actions overview](images/data-actions-overview.png)
+
+Estimated Time: __ minutes
```
Comment (william-masdon-oracle) [inline common-labs/data-actions/data-actions.md:9]: Please fill out how long this lab will take.

**Review by william-masdon-oracle — CHANGES_REQUESTED** [review]

File: common-labs/data-actions/data-actions.md (hunk 1 above)
Comment (NagwangGyamtso) [reply common-labs/data-actions/data-actions.md]: Done!

**Review by william-masdon-oracle — APPROVED** [review]

**Review by kamryn-v — APPROVED** [review]

**Pull request merged**